lib/gpu/Makefile.fermi

```diff
@@ -4,7 +4,7 @@ EXTRAMAKE = Makefile.lammps.standard
 CUDA_ARCH = -arch=sm_35
 CUDA_PRECISION = -D_SINGLE_DOUBLE
 CUDA_INCLUDE = -I$(CUDA_HOME)/include
-CUDA_LIB = -L$(CUDA_HOME)/lib64 -Xlinker -rpath -Xlinker $(CUDA_HOME)/lib64 -lcudart
+CUDA_LIB = -L$(CUDA_HOME)/lib64 -Xlinker -rpath -Xlinker $(CUDA_HOME)/lib64/stubs -lcudart
```

Comment:
This change is incorrect. You do **not** want to encode the CUDA stubs folder as rpath into an executable, you only want to use the stub folder for linking. At runtime you want the real libraries, not the stubs.

Reply:
You are right, sorry for the mistake.